crud/tarantool_test.go

```diff
@@ -0,0 +1,256 @@
+package crud_test
```

Comment:
Let's avoid integration tests as much as possible (I don't see the point in `crud` module testing). So let's break the tests into two parts:

## 1. Unit-tests

Here we could carefully check that the correct eval request is formed:
1. Create a code.<Any>Request with options.
2. Create expected tarantool.EvalRequest().
3. Compare results of Body()/Code()/Async().

+

Tests for Context()/Ctx():

https://github.com/tarantool/go-tarantool/blob/e52e9b228450d67b0e143a6ee7a95bd73a949855/request_test.go#L237-L302

## 2. Integration tests with Tarantool

It is enough to make 1 or 2 simple requests and check results for each <Any>Request.  There is no need to overcomplicate here because we already have tests for requests logic in the base package. As example, we don't need to check response existence in the subpackage if there is no error:

```Go
	if resp == nil {
		t.Fatalf("Response is nil after Select")
	}
```

Reply:
Thanks, done.